### ISSUE TITLE
OpenWatcom v2 build: compilation error on Ubuntu 24.10 (also with gcc 14.2.x)

### ISSUE THREAD

**Issue by LowLevelMahn** [open-watcom/open-watcom-v2#1340]
Ubuntu 24.10, gcc 14.2

```
cc guixloop.obj
../c/guixloop.c: In function ‘GUICreatePopup’:
../c/guixloop.c:227:38: error: passing argument 2 of ‘GUICreateMenuPopup’ from incompatible pointer type [-Wincompatible-pointer-types]
  227 |     ui_ev = GUICreateMenuPopup( wnd, &scr_location, wnd->menu, GUI_TRACK_LEFT, NULL );
      |                                      ^~~~~~~~~~~~~
      |                                      |
      |                                      guix_coord *
In file included from ../c/guixloop.c:40:
../../../../bld/gui/ui/h/guifloat.h:34:66: note: expected ‘guix_point *’ but argument is of type ‘guix_coord *’
   34 | extern ui_event GUICreateMenuPopup( gui_window *wnd, guix_point *scr_location, UIMENUITEM *menuitems,
      |                                                      ~~~~~~~~~~~~^~~~~~~~~~~~
Error(E42): Last command making (guixloop.obj) returned a bad status
Error(E02): Make execution terminated
<pmake -d build          -h> => non-zero return: 512
Build failed

```

**Comment by jmalak** [comment]
It looks like gcc 14 is more strict. Both structures are same but tag is different.
I will correct it, may be logical mistake.

**Comment by jmalak** [comment]
It is fixed by e6e8e8466c850ccdb788d662e81a094a601c6ac0 commit.
